src/app/app-routing.module.ts

```diff
@@ -4,7 +4,8 @@ import { RouterModule, Routes } from '@angular/router';
 const routes: Routes = [
   {
     path: '',
-    redirectTo: 'home'
+    redirectTo: 'home',
+    pathMatch: 'full'
```

Comment:
'pathMatch: full' we will use with empty-string path for prevent endless loop of routing.

For example. In first place in routing mostly is empty-string path with redirectTo, but when you want to go to other URL like /login, your array of routes are re-check and if spot empty-string is going to there once more and more. It is essensialy to know that, path always default is prefix. For prevent this endless loop, in object of your path, change pathmatch to full and it will be fix.